src/atoms/radio/Radio.spec.js

```diff
@@ -0,0 +1,96 @@
+import { mount } from '@vue/test-utils'
+import ARadio from './Radio.vue'
+
+describe('Radio', () => {
+  it('has default structure', () => {
+    const wrapper = mount(ARadio, {
+      propsData: {
+        id: '1',
```

Comment:
id not required so it's not part of default structure. Maybe we should set it as `required`?